exercises/Spatial-data-elevation-histogram-R.md

```diff
@@ -19,12 +19,12 @@ A colleague of yours is working on a project on banner-tailed kangaroo rats ([*D
 
 2. Clean up the data by:
 	* Using the `rename` function from `dplyr` to rename the second and third columns of this dataset to `longitude` and `latitude`
-	* Filter the data to only include those specimens with `Dipodomys_spectabilis.basisOfRecord` that is `PRESERVED_SPECIMEN` and a `Dipodomys_spectabilis.countryCode` that is `US`.
+	* Filter the data to only include those specimens with `Dipodomys_spectabilis.basisOfRecord` that is `PRESERVED_SPECIMEN` and a `Dipodomys_spectabilis.countryCode` that is `US`
 	* Remove points with values of `0` for `latitude` or `longitude`
-	* Use the `head()` function to show the top few rows of your cleaned up data
+	* Use the `head()` function and the `$` operator to show the top few rows of the Dipodomys_spectabilis.key column
```

Comment:
Yeah, it was because there's a lot of output due to the number of columns, and I thought trying to include all of that in the output figure, or just the end of it, would be confusing. What if we change it back to "Use the head() function..." and then add in the str() as another bullet point and have that be the solution figure?

Reply:
`str()` also produces a lot of output in the console.

Reply:
@ethanwhite what about if we have them `select` only the latitude and longitude columns as the last cleaning step in creating the dataframe, and then display the head of that?